sdk/maps/Azure.ResourceManager.Maps/api/Azure.ResourceManager.Maps.netstandard2.0.cs

```diff
@@ -215,4 +250,22 @@ public partial class MapsSku
         public static bool operator !=(Azure.ResourceManager.Maps.Models.MapsSkuName left, Azure.ResourceManager.Maps.Models.MapsSkuName right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct SigningKey : System.IEquatable<Azure.ResourceManager.Maps.Models.SigningKey>
```

Comment:
consider renaming to `MapsSigningKey` by adding it to prepend-rp-prefix in autorest.md since the naming is generic